content/blog/zardshard/2023-08-18_[gsoc_2023]_improving_icon-o-matic_final_report.md

```diff
@@ -0,0 +1,92 @@
++++
+type = "blog"
+title = "[GSoC 2023] Improving Icon-O-Matic Final Report"
+author = "Zardshard"
+date = "2023-08-18 11:21:50-04:00"
+tags = ["haiku", "software", "icon-o-matic", "gsoc", "gsoc2023"]
++++
+
+## What is Icon-O-Matic?
+
+![Screenshot of Icon-O-Matic](/files/blog/zardshard/icon_o_matic.png)
+
+There's a good chance that not everyone reading this article will know what Icon-O-Matic is, so I'll start by explaining what it is. Icon-O-Matic is a vector graphics editing program like Illustrator or Inkscape. It is specifically made to work with Haiku's custom HVIF vector graphics format. This format is similar to the SVG format, except optimized to be much, much smaller. The blog post ["500 Byte Images: The Haiku Vector Icon Format"](http://blog.leahhanson.us/post/recursecenter2016/haiku_icons.html) provides a more in-depth discussion for those interested.
+
+If you want to try Icon-O-Matic yourself, you will need to first install Haiku OS. Afterwards, you can launch it from the applications menu.
```

Comment:
```suggestion
If you want to try Icon-O-Matic yourself, you will need to first install Haiku. Afterwards, you can launch it from the applications menu.
```

(or you can say "the Haiku operating system" in case the clarification is needed)